_includes/manuals/nightly/3.6_upgrade.md

```diff
@@ -183,7 +183,23 @@ And on Debian/Ubuntu installations with Passenger, run:
 
     service apache2 restart
 
-#### Optional Step 5 - Cleanup of RPMs and repositories
+#### Optional Step 5 - Reclaim database space
+
+After database migrations, some space can sometimes be reclaimed. It's good idea to perform database vacuum for postgres on a regular basis as well rather than relying on autovacuum feature. For PostgreSQL do the following:
```

Comment:
Isn't "rather than relying on autovacuum feature" enough of a statement? What exactly do you want me to elaborate on? Doing the manual vacuum is still an option after migration which can drop some columns or indices.